OrbitClientServices/include/OrbitClientServices/ProcessClient.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CLIENT_SERVICES_PROCESS_CLIENT_H_
+#define ORBIT_CLIENT_SERVICES_PROCESS_CLIENT_H_
+
+//#include <chrono>
+//#include <memory>
+#include <string>
+
+#include "OrbitBase/Result.h"
+#include "grpcpp/grpcpp.h"
+//#include "module.pb.h"
+//#include "process.pb.h"
```

Comment:
nit: remove comments.

Reply:
Yes I realised after pushing and was updating this, thanks!